config/core/test-configs.yaml

```diff
@@ -2401,6 +2402,8 @@ test_configs:
   - device_type: sun50i-h6-pine-h64
     test_plans:
       - baseline
+      - kselftest-filesystems
+      - kselftest-seccomp
```

Comment:
Failing to DHCP in the kernel: https://lava.collabora.co.uk/scheduler/job/4246975

The network interface unique name needs to be defined in the device dictionary like for hana devices.

Reply:
Well no, that didn't work either with specific eth name:
https://lava.collabora.co.uk/scheduler/job/4260493
So this PR can be closed, these devices will require more work until they can run kselftest.